template_scripts/packages_fc.list

```diff
@@ -84,4 +86,4 @@ zip
 --exclude=vinagre
 --exclude=webkit2gtk3-plugin-process-gtk2
 --exclude=xorg-x11-drv-nouveau
---exclude=yum-utils
+--exclude=yum-utils
```

Comment:
Something is wrong with your editor configuration - it often truncates the final end of line marker.